app/config/api_platform/resources.yml

```diff
@@ -8,3 +8,4 @@ resources:
           id: '\d+'
         exceptionToStatus:
           PrestaShop\PrestaShop\Core\Domain\Hook\Exception\HookNotFoundException: 404
+          Lcobucci\JWT\Token\InvalidTokenStructure: 401
```

Comment:
It will only apply to `PrestaShopBundle\Api\Resource\HookStatus` and we don't want to repeat it everywhere